src/benchmark.rs

```diff
@@ -318,8 +341,10 @@ pub fn run_benchmark(
 
     // Determine number of benchmark runs
     let runs_in_min_time = (options.min_time_sec
-        / (res.time_real + prepare_res.time_real + shell_spawning_time.time_real))
-        as u64;
+        / (res.time_real
+            + setup_res.time_real
```

Comment:
I don't think `setup_res.time_real` should be added here. It's ran only once before each series of benchmarks, so it shouldn't influence how many benchmark runs we can perform in the "min. time" that is reserved for benchmarks.

Setup time will obviously be there in addition, but shouldn't influence how many benchmarks we perform.

Reply:
Note that the cleanup time is also not added here.